docs/concepts/structured_views.md

```diff
@@ -0,0 +1,39 @@
+# Concept: Structured Views
+
+Structured views are a type of [view](../concepts/views.md), which provide a way for developers using db-ally to define what they need from the LLM, including:
```

Comment:
maybe change "are a type of a view" into something more like -> Implements [View] interface, providing more controlled approach of using db-ally, including ?